src/components/Identifier/Identifier/Identifier.stories.tsx

```diff
@@ -0,0 +1,248 @@
+import React from 'react'
+/*  eslint-disable jsx-a11y/anchor-is-valid, react/jsx-key */
+import { Identifier } from './Identifier'
+import { IdentifierMasthead } from '../IdentifierMasthead/IdentifierMasthead'
+import { IdentifierLinks } from '../IdentifierLinks/IdentifierLinks'
+import { IdentifierGov } from '../IdentifierGov/IdentifierGov'
+import { IdentifierLinkItem } from '../IdentifierLinkItem/IdentifierLinkItem'
+import { IdentifierLink } from '../IdentifierLink/IdentifierLink'
+
+import dotGovIcon from 'uswds/src/img/icon-dot-gov.svg'
+
+export default {
+  title: 'Components/Identifier',
+  component: Identifier,
+  parameters: {
+    info: `
+  USWDS 2.0 Identifier component
+  
+  Source: https://designsystem.digital.gov/components/identifier/
+  `,
```

Comment:
We recently changed this convention for Storybook story descriptions. Could you update this one to match? See https://github.com/trussworks/react-uswds/commit/ed171f924c7455351f2d07a7422317191bd1f4bb for details.

Reply:
Updated